### PR TITLE
Update package p4est

### PR DESCRIPTION
**Pull request by gfcas**
Indention etc. Should be without any code change.

### CODE REVIEW

File: deal.II-toolchain/packages/dealii.package [hunk 1]
```diff
@@ -136,10 +135,8 @@ fi
 
 if [ ! -z "${P4EST_DIR}" ]; then
     cecho ${INFO} "deal.II: configuration with P4EST"
-    CONFOPTS="\
-        ${CONFOPTS} \
-        -D DEAL_II_WITH_P4EST:BOOL=ON \
-        -D P4EST_DIR=${P4EST_DIR}"
```
Comment (koecher) [inline deal.II-toolchain/packages/dealii.package:142]: don't we need this anymore?

Comment (koecher) [reply deal.II-toolchain/packages/dealii.package:142]: Precisely: Line 142 `-D P4EST_DIR=${P4EST_DIR}"`

Comment (gfcas) [reply deal.II-toolchain/packages/dealii.package:142]: Oh yeah, this is not needed any more. Thanks!

Comment (gfcas) [reply deal.II-toolchain/packages/dealii.package:142]: @koecher sorry, you are absolutely right, this was deleted by a too fast rebase... let me add this again.... (I was at an other line where I exactly removed such a case)...

File: deal.II-toolchain/packages/dealii.package [hunk 2]
```diff
@@ -136,10 +135,9 @@ fi
...
+    CONFOPTS="${CONFOPTS} \
+      -D DEAL_II_WITH_P4EST:BOOL=ON"
+      -D P4EST_DIR=${P4EST_DIR}"
```
Comment (tjhei) [inline deal.II-toolchain/packages/dealii.package:140]: This doesn't look right. Where is the \?

Comment (gfcas) [reply deal.II-toolchain/packages/dealii.package:140]: Yeah sorry ....

**Comment by gfcas** [comment]
This should now the version, which works. However, I suppose that it is in fact not necessary to set the Package_DIR, as after each installed package a variable PACKAGE_DIR is exported and dealii see this, or am I totally wrong?

**Review by tjhei — APPROVED** [review]

File: deal.II-toolchain/packages/dealii.package (hunk 2 above)
Comment (koecher) [reply deal.II-toolchain/packages/dealii.package:140]: and the additional trailing quote in 139 shouldn't be there

**Comment by tjhei** [comment]
Good to merge once tests pass.

File: deal.II-toolchain/packages/dealii.package (hunk 2 above)
Comment (koecher) [reply deal.II-toolchain/packages/dealii.package:140]: Ah, was resolved already

**Comment by gfcas** [comment]
Extra CI on p4est+dealii is running under https://github.com/gfcas/candi/actions/runs/983195015

**Comment by koecher** [comment]
The osx tester fails are unrelated to p4est.

**Comment by gfcas** [comment]
> 
> 
> The osx tester fails are unrelated to p4est.

OK, thanks!

**Pull request merged**